devutils/src/devutils/invoke_utils.py

```diff
@@ -197,15 +200,31 @@ def _make_conda_env_if_needed():
                 )
 
     @task
-    def install(ctx):
-        """Install dependencies"""
-        _make_conda_env_if_needed()
-        poetry(ctx, "install")
+    def install(ctx, local: str = None, update: bool = False, verbose: bool = False):
+        """
+        Installs or updates dependencies.
 
-    @task
-    def devinstall(ctx):
```

Comment:
Where did `devinstall` go? It should still be there...

Reply:
We delete it in favor of --local argument on install. But if you think we need a standalone command we can put it back.

Reply:
I'd rather have a standalone command -- and then you won't need to update all the github actions to use the new version too ;)

Reply:
ok, I'll add it back

Reply:
Good point!